tools/wake/main.cpp

```diff
@@ -594,8 +594,16 @@ int main(int argc, char **argv) {
 
   bool enumok = true;
   std::string libdir = wcl::make_canonical(find_execpath() + "/../share/wake/lib");
-  auto wakefilenames =
-      find_all_wakefiles(enumok, clo.workspace, clo.verbose, libdir, ".", user_warn);
+  std::vector<std::string> wakefilenames;
+  {
+    auto start = std::chrono::steady_clock::now();
+    wakefilenames = find_all_wakefiles(enumok, clo.workspace, clo.verbose, libdir, ".", user_warn);
+    auto stop = std::chrono::steady_clock::now();
+    auto duration = std::chrono::duration_cast<std::chrono::microseconds>(stop - start).count();
+    wcl::log::info("Find all wakefiles took %f seconds",
+                   static_cast<double>(duration) / 1000000.0)();
```

Comment:
nit (And I stress this is a *nit*, you can land this as is):

```
    auto duration = std::chrono::duration_cast<std::chrono::seconds, double>(stop - start).count();
    wcl::log::info("Find all wakefiles took %f seconds", duration)();
```

Reply:
its trivial to  fix, and strictly better. No reason to not just fix it